### PR TITLE
Windows: correctly set notifications name - not electron.app.YOURAPPNAME

### PR DESCRIPTION
**Pull request by si458**
this helps with setting the the name correctly for windows notifications
currently it shows as `electron.app.YOURAPPNAME`
you need to run this command aswell as starting the app from the start menu or shortcut
https://www.electronjs.org/docs/latest/tutorial/notifications#windows

### CODE REVIEW

File: app/src/main.ts [hunk 1]
```diff
@@ -69,6 +69,8 @@ if (!appArgs.userAgentHonest) {
   }
 }
 
+app.setAppUserModelId(app.getName());
```
Comment (ronjouch) [inline app/src/main.ts:72]: Two things:

1. Pls wrap in an `if (isWindows())` . We have such a helper function somewhere.
2. Comment, pls, and link to https://www.electronjs.org/docs/latest/api/app#appsetappusermodelidid-windows

**Review by ronjouch — COMMENTED** [review]
Neat, thanks! Just two nits and I'll merge this.

**Comment by si458** [comment]
> Neat, thanks! Just two nits and I'll merge this.

I think I've done it correctly if you can verify please?

**Comment by ronjouch** [comment]
> > Neat, thanks! Just two nits and I'll merge this.
> 
> I think I've done it correctly if you can verify please?

@si458 perfect. Merging, will ship with the next release, in a few days/weeks. Thanks!

**Comment by AgentSmith0** [comment]
Is there any way to fix https://github.com/nativefier/nativefier/issues/1253 like this PR does?

**Pull request merged**